### PR TITLE
Trev8939/display kml

### PR DESCRIPTION
**Pull request by TADraeseke**
As per: https://devtopia.esri.com/runtime/common-samples/issues/874

Any chance you have the scope to review this for me please @eri9600?

### CODE REVIEW

File: java/display-kml/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Display KML
+Display a KML layer from a URL, portal item, or local KML file.
+
+![Display KML App](display-kml.png)
+
+## How to use the sample
+Select the KML layer you'd like to display from the dropdown menu in the upper right
```
Comment (elopezsolis) [inline java/display-kml/README.md]: Maybe you want to let the user know to download the file first and adb it to the device before they click on "KML from internal storage".  You can do something like this `Download file first, see [Offline Data](#offline-data)`

File: java/display-kml/src/main/res/menu/kml_sources.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/kmlFromUrl"
+        android:title="@string/kml_from_url" />
+
+    <item
+        android:id="@+id/kmlfromPortal"
```
Comment (elopezsolis) [inline java/display-kml/src/main/res/menu/kml_sources.xml]: capitalize "From"

File: java/display-kml/README.md (hunk 1 above)
Comment (TADraeseke) [reply java/display-kml/README.md]: Good idea!

**Comment by TADraeseke** [comment]
Thanks @eri9600!

**Pull request merged**